README.md

```diff
@@ -2,21 +2,54 @@
 
 ## Build site
 
-To build the website locally, clone the repo with:
+To build the website locally, clone the repo and change into the repo's directory.
 
 ```
 git clone https://github.com/blab/blotter.git
+cd blotter/
 ```
 
-Then install necessary Ruby dependencies by running `bundle install` from within the `blotter` directory.  After this, the site can be be built with:
+Install the latest version of Ruby supported by blotter using [rbenv](https://github.com/rbenv/rbenv) to work around any system-wide Ruby installations that may be incompatible.
+
+``` bash
+rbenv install 3.2.6
+```
+
+Set a local Ruby version for the blotter directory.
+
+``` bash
+rbenv local 3.2.6
+```
```

Comment:
As an alternative approach — you could remove `.ruby-version` from `.gitignore`, include `.ruby-version` in this commit (with contents of "3.2.6"), and then these instructions reduce to `rbenv install` — it will install the version that's in the `.ruby-version` file, and you won't have to explicitly activate it, the `rbenv` shims will DTRT. 

I have a _slight_ preference for documenting the version in the `.ruby-version` file in this way, because when it becomes necessary to upgrade, there's a single thing that has to change, and as folks pull in new changes, they will discover the need to install the new Ruby at that point.

Reply:
I didn't know about this approach, but I like it. Trying it out now from my home computer, but first I have to reinstall rbenv for some reason...

Reply:
This worked really well, @genehack. Thanks again for the recommendation and review!